fuzz/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+
+[package]
+name = "parsec-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2018"
```

Comment:
Do we care about filling those with the same values as in our other crates?

Reply:
wasn't sure about that... it was obviously automatically generated, I can update it to be something similar to the parsec `Cargo.toml`

Reply:
I think it would be good

Reply:
I'll only update the authors, will leave version as "0.0.0"